contest/templates/contest/contestArchive.html

```diff
@@ -17,6 +17,25 @@
   <div class="well">
     <h2>Contest Archive</h2>
   </div>
+  {% if user.is_authenticated %}
+  <div class="btn-group" role="group" style="float:right;">
+    {% if user.has_subjudge_auth %}
```

Comment:
Sub judges maybe attend some contests.
So you have to merge owned and attended contests for them.